common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -739,6 +789,24 @@ def get_transcripts_info(self, is_bumper=False):
             language_code: transcript_file
             for language_code, transcript_file in transcripts.items() if transcript_file != ''
         }
+
+        # For phase 2, removing `include_val_transcripts` will make edx-val
+        # taking over the control for transcripts.
+        if include_val_transcripts:
+            transcript_languages = get_available_transcript_languages(
+                edx_video_id=self.edx_video_id,
+                youtube_id_1_0=self.youtube_id_1_0,
+                html5_sources=self.html5_sources
+            )
+            # HACK Warning! this is temporary and will be removed once edx-val take over the
+            # transcript module and contentstore will only function as fallback until all the
+            # data is migrated to edx-val.
+            for language_code in transcript_languages:
+                if language_code == 'en' and not sub:
+                    sub = NON_EXISTENT_TRANSCRIPT
+                elif not transcripts.get(language_code):
+                    transcripts[language_code] = NON_EXISTENT_TRANSCRIPT
```

Comment:
Can we add some explanation here why we are setting these dummy values?

Reply:
Shouldn't we add tests for these dummy values?

Reply:
they are added.